### ISSUE TITLE
Giving away visor headband polypropylene to people/schools who have laser cutters

### ISSUE THREAD

**Issue by ajlennon** [DoESLiverpool/covid19#72]
As we are now giving away visors and as has been pointed out to me numerous times we have **lots** of polypropylene for visor headbands can we start to give some of this to people with laser cutters who are asking for materials so they can make visors for the NHS? For simplicity of supply their bands would then be combined with die-cut visor shields from 3DCrowd and go to fulfil 3DCrowd orders.

@JackiePease @amcewen @afroleft @mjamjoum ?

**Comment by plastictactics** [comment]
Lydiate schools trust would like material, but only to make their design. Giving them material would result in more visors making their way to the frontline quicker, so makes sense to me as long as it can be accounted for. But have made clear to them that it's not my decision - have asked the question a few times directly and on github and appreciate you raising it @ajlennon.

**Comment by JimNixon** [comment]
I've got a 120w 60x90cm epilog laser in Woolton that I could run a batch of
visor headbands on.

**Comment by ajlennon** [comment]
@plastictactics  Yeah this is one of the main conundrums for me. I don't know what the answer is. My feeling is that in terms of polyprop it's as Adrian has said -  we have **loads**. That didn't bother me when we ordered as I fully expected we would be providing materials to others with making capability. And of course that's exactly what's happening. In terms of whether we give away or not I'm not sure. One model that was suggested to me by local schools who want materials is they take materials and give us a percentage of the visors they make. That sort of makes sense? I have no clear view tbh atm

**Comment by ajlennon** [comment]
My thinking has been that for awesome people like @JimNixon we can ask him to pick up polyprop from us and deliver visor bands to us. Perhaps Coffee_Dave can help with logistics

**Comment by ajlennon** [comment]
Also we have Dan at Sensor City who will be doing some cutting for us and need materials. I can get those to him

**Comment by JimNixon** [comment]
> 
> 
> My thinking has been that for awesome people like @JimNixon we can ask him to pick up polyprop from us and deliver visor bands to us. Perhaps Coffee_Dave can help with logistics

I don't have access to a car at the moment but might be able to call in a favor to get a mate to drop off the plastic at mine. Happy to cut as much polyprop as you can give me as I imagine my laser is about 2 or 3 times quicker than Gerald.

**Comment by ajlennon** [comment]
Hi @JimNixon - or alternatively depending on where you are I could drop some around? I have to deliver some to Sensor City tomorrow morning anyway so...

**Comment by JimNixon** [comment]
I'll be in all day tomorrow, so whenevers best for you. Sent you a DM on
Twitter with my address, didn't want to post it online.

**Comment by ajlennon** [comment]
Thanks JIm - sure I'll bring some over then. Any thoughts on how many sheets?

**Comment by JimNixon** [comment]
How many sheets are Gerald going through each day? Got time to do 2 days
worth of cutting to start with.

**Comment by ajlennon** [comment]
@JimNixon so the process keeps being tweaked but I think we're currentlly doing 20 pieces per sheet and that's taking 20-25 minutes on Gerald.

So if you can have a think about how many hours you have and let me know overall I'll get the sheets over to you.

Thanks !!!

**Comment by JimNixon** [comment]
Could spare about 12 hours over the next few days so I'll start off with 40
sheets if you've got that many.

**Comment by ajlennon** [comment]
No problems with me taking 40 to @JimNixon and 10 to Dan @ SC are there @amcewen @afroleft @mjamjoum ?
